addons/web/static/src/webclient/actions/debug_items.js

```diff
@@ -72,6 +72,27 @@ function viewFields({ action, env }) {
     };
 }
 
+function ViewModel({ action, env }) {
+    if (!action.id) {
+        return null;
+    }
+    const model_name = action.res_model
+    const description = _t("View Model: ") + model_name;
+    return {
+        type: "item",
+        description,
+        callback: async () => {
+            const modelId = (
+                await env.services.orm.search("ir.model", [["model", "=", action.res_model]], {
```

Comment:
```suggestion
                await env.services.orm.search("ir.model", [["model", "=", modelName]], {
```

Reply:
Hello @aab-odoo ,

I have made the changes, Could you please take a look at it?

Thank You🙂